extensions/liquibase/runtime/src/main/java/io/quarkus/liquibase/LiquibaseFactory.java

```diff
@@ -64,7 +64,11 @@ public Liquibase createLiquibase() {
                     database.setDefaultSchemaName(config.defaultSchemaName.get());
                 }
             }
-            return new Liquibase(config.changeLog, resourceAccessor, database);
+            Liquibase liquibase = new Liquibase(config.changeLog, resourceAccessor, database);
+            if (config.params != null) {
+                config.params.forEach((k, v) -> liquibase.getChangeLogParameters().set(k, v));
```

Comment:
We avoid lambdas in runtime code as they have a cost.

The map should never be null.